torch/ao/quantization/utils.py

```diff
@@ -183,11 +183,22 @@ def weight_dtype(qconfig):
     weight = qconfig.weight()
     return weight.dtype
 
+# Note: the type is Optional[bool] to match backend_config_dict
+def is_dynamic(qconfig) -> Optional[bool]:
+    assert qconfig is not None
+    activation = qconfig.activation()
+    if hasattr(activation, 'compute_dtype'):
+        return True
+    return None
```

Comment:
Can we return `False` instead? To me it is harder to understand and there is a subtle difference between`if is_dynamic(...) is None` vs `if is_dynamic(...)`

Reply:
this matches existing backend_config_dict code, let's split cleaning this up in a separate PR